vagrant_files/setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+# Docker setup
+echo "Installing docker"
+yum install -y docker
```

Comment:
Looks like this step is failing on `vagrant up`

```
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'centos70' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: >= 0
==> default: Adding box 'centos70' (v0) for provider: virtualbox
    default: Downloading: http://deltaforce.io/box/centos70-base_1414068169.box
==> default: Successfully added box 'centos70' (v0) for 'virtualbox'!
==> default: Importing base box 'centos70'...
==> default: Matching MAC address for NAT networking...
==> default: Setting the name of the VM: elasticsearch_default_1428917354560_77107
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 22 => 2222 (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
==> default: Mounting shared folders...
    default: /vagrant => /Users/mwl/IdeaProjects/es-mesos/elasticsearch
==> default: Running provisioner: shell...
    default: Running: /var/folders/cg/d8505chj7pz0hbqgd74xvsyh0000gn/T/vagrant-shell20150413-21722-grgzbi.sh
==> default: Installing docker
==> default: Loaded plugins: fastestmirror
==> default:
==> default:
==> default:  One of the configured repositories failed (Unknown),
==> default:  and yum doesn't have enough cached data to continue. At this point the only
==> default:  safe thing yum can do is fail. There are a few ways to work "fix" this:
==> default:
==> default:      1. Contact the upstream for the repository and get them to fix the problem.
==> default:
==> default:      2. Reconfigure the baseurl/etc. for the repository, to point to a working
==> default:         upstream. This is most often useful if you are using a newer
==> default:         distribution release than is supported by the repository (and the
==> default:         packages for the previous distribution release still work).
==> default:
==> default:      3. Disable the repository, so yum won't use it by default. Yum will then
==> default:         just ignore the repository until you permanently enable it again or use
==> default:         --enablerepo for temporary usage:
==> default:
==> default:             yum-config-manager --disable <repoid>
==> default:
==> default:      4. Configure the failing repository to be skipped, if it is unavailable.
==> default:         Note that yum will try to contact the repo. when it runs most commands,
==> default:         so will have to try and fail each time (and thus. yum will be be much
==> default:         slower). If it is a very temporary problem though, this is often a nice
==> default:         compromise:
==> default:
==> default:             yum-config-manager --save --setopt=<repoid>.skip_if_unavailable=true
==> default:
==> default: Cannot find a valid baseurl for repo: base/7/x86_64
==> default: Could not retrieve mirrorlist http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=os&infra=stock error was
==> default: 14: curl#6 - "Could not resolve host: mirrorlist.centos.org; Name or service not known"
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```